pkg/asset/manifests/dns.go

```diff
@@ -69,8 +71,9 @@ func (d *DNS) Generate(dependencies asset.Parents) error {
 			BaseDomain: installConfig.Config.ClusterDomain(),
 		},
 	}
+	platformName := installConfig.Config.Platform.Name()
 
-	switch installConfig.Config.Platform.Name() {
+	switch platformName {
```

Comment:
why this change?

Reply:
for readability

Reply:
hmm, not sure how it is more readable, your change makes more sense if platformName has consumers outside the switch. which currently there aren't. so the previous state makes more sense.

Reply:
reverted